Mobile/iOS/Templates/Release_3.4.7/xxx-Federate_sharing.md

```diff
@@ -0,0 +1,55 @@
+| **Errors** ||||||
+| 13 |  Inexistent User |  1. Repeat the test cases 2. or 3. using a username which does not exist in remote server  | The file can not be shared, an error appears |||
+| 14 |  Inexistent Server |  1. Repeat the test cases 2. or 3. using an inexistent remote URL | The file can not be shared, an error appears |||
+| 15 |  Share twice fed |  1. Share federated twice with the same user | First time the file is shared, second time, an error is received |||
+| 16 |  Deleted account |  1. Repeat the test cases 2. or 3.<br> 2. From admin in web, delete the account ot the sharing user | The sharee can not view the shared file or folder |||
+| 17 |  Without connection |  1. Switch the wifi connection off<br>2. Try to share with a federated user | An error appears |||
+| 18 |  Without server connection |  1. Switch the server off<br>2. Try to share with a federated user | An error appears |||
+| **Server Capabilities** ||||||
+| 19 |  Federated Sharing disabled |  1. In server side, disable the capability to share with federation<br>2. In app, try to share with federation<br> | The share is not accepted and an error is raised |||
+| 20 |  Receive Federated disabled |  1. In server side, disable the capability to share with federation<br>2. In app, try to share form a other server's user with the current server<br> | The share is not accepted and an error is raised |||
+| **Edit Privilege** ||||||
+| 21 | Edit granted file|  1. Swipe to share a file with users<br>2. Type a federated URL<br>3. Grant the federated user to edit<br> 4. Login in web with the sharee and acept the federated share<br>5. Login in app with the sharee and try to edit the file  | The shared file appears in files view and can be edited |||
+| 22 | Edit granted folder|  1. Swipe to share a folder with users<br>2. Type a federated URL<br>3. Grant the federated user to edit<br> 4. Login in web with the sharee and acept the federated share<br>5. Login in app with the sharee and try to edit the folder  | The shared folder appears in files view and can be edited (upload/update/delete) |||
+| 23 | Edit not granted file |  1. Swipe to share a file with users<br>2. Type a federated URL<br>3. Do not grant the federated user to edit<br> 4. Login in web with the sharee and acept the federated share<br>5. Login in app with the sharee and try to edit the file  | The shared file appears in files view and can not be edited |||
+| 24 | Edit not granted folder |  1. Swipe to share a folder with users<br>2. Type a federated URL<br>3. Do not grant the federated user to edit<br> 4. Login in web with the sharee and acept the federated share<br>5. Login in app with the sharee and try to edit the folder  | The shared file appears in files view and can not be edited (upload/update/delete)|||
+| **Reshare Privilege** ||||||
```

Comment:
Right. Related with the re-sharing i've added more test cases taking in account the different version crossing in the federating shares. But all is pending of the core issue.